src/Components/InputSearchField/InputSearchField.tsx

```diff
@@ -0,0 +1,59 @@
+// import { Input } from '@mui/material';
+import React from 'react';
+
+// const InputSearchField = () => {
+//   return (
+//     <Input aria-label="Demo input" placeholder="Type something…" />
+//   );
+// }
+
+// export default InputSearchField;
+
+import { Input as BaseInput } from '@mui/base/Input';
+import { styled } from '@mui/system';
```

Comment:
avoid the use of styled-components, use CSSmodules instead to reduce performance overheads, bundle size and load complexity